examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -12246,6 +12432,57 @@
               "maxInterval": 65344,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "SwitchLatched",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "InitialPress",
```

Comment:
This does not match the feature map (which claims this is a latching switch).  We should probably not enable things that make this obviously not spec-compliant....

Reply:
fixed